orchestrator/ethereum_oracle/src/ethereum_event_watcher.rs

```diff
@@ -7,7 +7,7 @@ use utils::{
     error::OrchestratorError,
     types::{
         UbtDeposited,
-        ValidatorPowerChangeEvent,
+        PayeeUpdated,
```

Comment:
Maybe this naming is to cryptic, i like the old one better. Something on the lines of ValidatorStakingPowerChanged...?

Reply:
that would mean we have to change event name in solidity contract as well, and there we already use payee names, so we have to change payee to validator in whole contract, if we decide to do it, lets do separate task maybe
this event maps to MsgValidatorPowerChangedClaim transaction anyways